docs/List.md

```diff
@@ -2198,6 +2199,31 @@ const PostList = props => (
 )
 ```
 
+### `isRowExpandable`
+
+You can customize which rows will allow to show an expandable panel below them using the `isRowExpandable` prop. It expects a function that will receive the record of each `<DatagridRow>` and returns a boolean expression.  For instance, this code shows an expand button only for rows with an id greater than 300:
```

Comment:
the example (id > 300) isn't very representative of a real-world problem. Please look for a better example.

Reply:
Ok